### DISCUSSION TITLE
Slider tick counter doesn't show the max correctly on the results screen

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#25463] [Q&A] [answered]
I don't know if it's intended to work this way for now, but after the last update the slider tick max on results screen is always equal to the achieved slider ticks number. Here is two results that I achieved on the same beatmap difficulty:
![image](https://github.com/ppy/osu/assets/53406701/58f73ee6-d00f-49a7-935e-6d066e117b87) ![image](https://github.com/ppy/osu/assets/53406701/2e0e4fa7-9961-4389-91b2-c3066c99f647)

**Comment by peppy** [comment, marked as answer]
Tracking at https://github.com/ppy/osu/issues/25464.

**Comment by DarkbitNetwork** [comment]
From my observation, slider ends are counted as slider ticks in the score card (or result screen), so missing slider ends will reduce the **maximum** slider ticks counter instead of reducing the achieved slider tick counter. Missing the actual slider tick still reduces the achieved slider tick counter though.
